graph/schema/event/event.graphql

```
@@ -42,15 +42,16 @@ type Response{
 
 type Query {
   events: [Event!]!
+  myEvents: [Event!]! @shouldBeAuthenticated
   event(id: ID!): Event!
-  getMembersOfEvent(id:ID!):[Member!]!
+  getMembersOfEvent(id:ID!):[Member!]! @shouldBeAuthenticated
 }
 
 type Mutation {
-  createEvent(data: EventInput!): EventResponse!
-  addMembersToEvent(id:ID!,data: AddMemberInput!): String!
-  removeMemberFromEvent(id:ID!,memberId:String!): Response!
-  deleteEvent(id: ID!): Response!
-  updateEvent(id: ID!, data: EventInput!): EventResponse!
-  updateSchedule(id: ID!, data: ScheduleUpdateInput!): Response!
+  createEvent(data: EventInput!): EventResponse! @shouldBeAuthenticated
+  addMembersToEvent(id:ID!,data: AddMemberInput!): String! @shouldBeAuthenticated
+  removeMemberFromEvent(id:ID!,memberId:String!): Response! @shouldBeAuthenticated
```

Rather than generic Response with boolean flag for success, it's better to return a formatted error when error occurs. something like
```json
{
  "errors": [
    {
      "message": "Cannot query field \"__typenam\" on type \"Query\".",
      "locations": [
        {
          "line": 1,
          "column": 2
        }
      ],
      "extensions": {
        "code": "INVALID_INPUT",
        "stacktrace": [
          "GraphQLError: Cannot query field \"__typenam\" on type \"Query\".",
          "    at Object.Field (/my_project/node_modules/graphql/validation/rules/FieldsOnCorrectTypeRule.js:48:31)",
          "    ...additional lines..."
        ]
      }
    }
  ]
}
```